packages/amplify-e2e-core/src/index.ts

```diff
@@ -111,14 +112,16 @@ export async function createNewProjectDir(
   // eslint-disable-next-line spellcheck/spell-checker
   prefix = path.join(fs.realpathSync(os.tmpdir()), amplifyTestsDir),
 ): Promise<string> {
-  const currentHash = execSync('git rev-parse --short HEAD', { cwd: __dirname }).toString().trim();
   let projectDir;
   do {
-    const randomId = await global.getRandomId();
-    projectDir = path.join(prefix, `${projectName}_${currentHash}_${randomId}`);
+    projectDir = path.join(prefix, `${projectName}_${Math.floor(Math.random() * 1000000)}`);
```

Comment:
this became more important as we started running multiple tests concurrently; long project names would overlap because the first 20 characters generated by this could be identical - due to the commit hashes being identical (currentHash)

The primary change here is that random number is moved to the 2nd portion of the directory name, instead of the last portion of it.